docs/UserGuide.md

```diff
@@ -52,6 +53,31 @@ You will see a message indicating successful addition of the musician like below
 Should you input a musician which is already in your contact book(ie. have the same phone number or email as an existing contact), you will see an error message showing the possible error, please input a different phone/email for the current contact or modify the original contact's relevant details.
 
 
+### Tagging a musician with genres: `tag genre`
+
+Tags a musician with one or more genres he/she is proficient in.
+
+**Format:** `tag genre INDEX g/GENRE...`
+
+**Constraints:**
+* `INDEX` must be a positive integer 1, 2, 3, …​
+* The genre tag must be *non-empty*.
+* You have to supply *at least one genre tag* to the musician you are tagging.
+
+**Examples:**
+* `tag 1 g/rock g/pop`
```

Comment:
Nit: maybe you can be consistent with the format you described earlier, as in `tag genre 1 g/rock g/pop`?